### DISCUSSION TITLE
Is bicycle_road=yes supported?

### DISCUSSION THREAD

**Discussion by xephos1one** [organicmaps/organicmaps#5185] [Q&A] [answered]
Hi,
is bicycle_road=yes supported and taken into account as a preferred route for bikes?
https://wiki.openstreetmap.org/wiki/Tag:bicycle_road%3Dyes
From my experience currently secondary is preferred over residential with bicycle_road=yes when it should be the other way around.
I haven't seen the tag mentioned anywhere in this Repo yet so I thought to ask here.

**Comment by vng** [comment, marked as answer]
Hm, no. Should add bicycle_road=yes and cyclestreet=yes.

**Comment by carlostipton1319** [comment]
Yes, the tag "bicycle_road=yes" is supported in OpenStreetMap (OSM) and commonly used to indicate that a specific road is designated for bicycle use. This tag helps in providing accurate information about cycling  [is it worth ](https://thetraveltourism.com/ado-a20-electric-bike-is-it-worth-it/)infrastructure and assists cyclists in route planning. It is important to note that the availability and usage of this tag may vary depending on the specific mapping community and region.
